cw-orch-daemon/src/core.rs

```diff
@@ -51,13 +49,13 @@ use tonic::transport::Channel;
     ## Querying
 
```

Comment:
Nit: The comments above with `TxHandler` are incorrect for this struct.

Reply:
> Nit: The comments above with `TxHandler` are incorrect for this struct.

Aha, wrong path and it doesn't implement TxHandler. Updated it!